pkg/kubernetes/mtls.go

```diff
@@ -165,7 +165,10 @@ func Expiry() (*time.Time, error) {
 		return nil, err
 	}
 
-	caCrt := secret.Data["ca.crt"]
+	caCrt, ok := secret.Data["ca.crt"]
+	if !ok {
+		return nil, errors.New("root certificate not loaded yet, please try again in few minutes")
```

Comment:
@yaron2 This issue was found on this code when tested on windows. 
When dapr is installed on Kubernetes and quickly `dapr mtls expiry` is run then there was a panic seg fault on line 173 since it takes few seconds for the caCert to be loaded to the k8s store after init. 
This is a fix for that. Please review ...